web_company_color/models/res_company.py

```diff
@@ -196,6 +198,7 @@ def _scss_get_sanitized_values(self):
                 or 'theme-color("primary")',
                 "color_link_text_hover": values.get("color_link_text_hover")
                 or 'darken(theme-color("primary"), 10%)',
+                "color_input_bg": values.get("color_input_bg") or "#FFF",
```

Comment:
Use the default color for required fields. Unfortunately the color value is hardcoded :/
```suggestion
                "color_input_bg": values.get("color_required_input_bg") or "#D2D2FF",
```